app/page.jsx

```diff
@@ -1,35 +1,61 @@
-import React from "react";
-import TicketCard from "./(components)/TicketCard";
+"use client"
+import { useState, useEffect } from "react";
+import TicketCard from "./(components)/TicketCard"; // Adjust the import according to your file structure
 
-const getTickets = async () => {
```

Comment:
why do you believe `getTickets` is required to be declared inside `useEffect`?

Reply:
If there is any update in ticket database that should reflect in the application in sync with it so I believe that featching the tickets every time the page renders keep the frontend updated

Reply:
that could have been achieved without introducing this change, the getTicket is already called each time the we hit the dashboard
it would be nice if you can revert this one

Reply:
But wouldn't use effect would run it as a side effect improve our site's proformance (just being curious)